x-pack/plugins/apm/public/components/app/service_inventory/index.tsx

```diff
@@ -180,6 +180,7 @@ export function ServiceInventory() {
     !userHasDismissedCallout;
 
   const isLoading = mainStatisticsStatus === FETCH_STATUS.LOADING;
+  const errorFetching = mainStatisticsStatus === FETCH_STATUS.FAILURE;
```

Comment:
Can we call this something that indicates it's a boolean and perhaps is more consistent with `isLoading`:

```suggestion
  const isFailure = mainStatisticsStatus === FETCH_STATUS.FAILURE;
```

Reply:
Agree is a better name for what is it.